hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -498,6 +498,26 @@ int _imxrt_setDevClock(int clock, int div, int mux, int mfd, int mfn, int state)
 }
 
 
+int _imxrt_setDirectLPCG(int clock, int on)
```

Comment:
In our code, the second argument of such functions usually is named as `val` or `state` (based on API for imxrt and stm). If it is also a common practice I am ok with that so that up to you.

Reply:
Changed to `state`, which should indicate whether the clock gate is turned `on` or `off`.